go.mod

```
@@ -104,6 +81,35 @@ require (
 	k8s.io/utils v0.0.0-20221128185143-99ec85e7a448
 )
 
+require (
+	// open telemetry dependencies
+	// pinned to v0.97.0 due to issue with ports binding which causes our tests to fail
+	// ref: https://github.com/open-telemetry/opentelemetry-collector/issues/10031
```

Nit: use more action-oriented language so we know when its OK to remove the pinning.

```suggestion
	// Do NOT upgrade the version beyond 0.97.0 until https://github.com/open-telemetry/opentelemetry-collector/issues/10031 is fixed!
```